README.md

```diff
@@ -45,26 +30,58 @@ Jazzer currently supports the following platforms:
+
+### Maven
+
+Add the following to your `pom.xml`:
+
+```xml
+<dependency>
+    <groupId>com.code-intelligence</groupId>
+    <artifactId>jazzer-junit</artifactId>
+    <version>LATEST VERSION</version>
+</dependency>
+```
+
+A complete example project using Maven is available in [`examples/junit`](examples/junit).
+
+### Gradle
 
-### JUnit 5
+Include Jazzer in your `build.gradle`:
 
-The following steps assume that JUnit 5.9.0 or higher is set up for your project, for example based on the official [junit5-samples](https://github.com/junit-team/junit5-samples).
+```gradle
+implementation group: 'com.code-intelligence', name: 'jazzer-junit', version: '<LATEST VERSION>'
+```
+
+### Bazel
+
+Jazzer is supported via [rules_fuzzing](https://github.com/bazelbuild/rules_fuzzing), the official Bazel rules for fuzzing.
+For setup instructions, see [the README](https://github.com/bazelbuild/rules_fuzzing#java-fuzzing).
+
+With Jazzer set up, you can start writing fuzz tests and benefit from automated bug discovery and improved code coverage.
 
-1. Add a dependency on `com.code-intelligence:jazzer-junit:<latest version>`.
-   All Jazzer Maven artifacts are signed with [this key](deploy/maven.pub).
-2. Add a new *fuzz test* to a new or existing test class: a method annotated with [`@FuzzTest`](https://codeintelligencetesting.github.io/jazzer-docs/jazzer-junit/com/code_intelligence/jazzer/junit/FuzzTest.html) and at least one parameter.
-   A list of supported parameter types can be found in the [documentation](docs/junit-integration.md#supported-types).
-3. Assuming your test class is called `com.example.MyFuzzTests`, create the *inputs directory* `src/test/resources/com/example/MyFuzzTestsInputs`.
-4. Run a fuzz test with the environment variable `JAZZER_FUZZ` set to `1` to let the fuzzer rapidly try new sets of arguments.
-   If the fuzzer finds arguments that make your fuzz test fail or even trigger a security issue, it will store them in the inputs directory.
-   In this mode, only a single fuzz test is executed per test run (see [#599](https://github.com/CodeIntelligenceTesting/jazzer/issues/599) for details).
-5. Run the fuzz test without `JAZZER_FUZZ` set to execute it only on the inputs in the inputs directory.
-   This mode, which behaves just like a traditional unit test, ensures that issues previously found by the fuzzer remain fixed and can also be used to debug the fuzz test on individual inputs.
+## Writing fuzz tests
 
-A simple property-based fuzz test could look like this:
+To write a fuzz test, add a method to your test class and annotate it with [`@FuzzTest`](https://codeintelligencetesting.github.io/jazzer-docs/jazzer-junit/com/code_intelligence/jazzer/junit/FuzzTest.html).
+Jazzer will automatically generate and mutate inputs for your method parameters.
+You can use primitives, strings, arrays, and many standard library classes—see the [mutation framework documentation](docs/mutation-framework.md#supported-types) for details.
+
```

Comment:
I would like to add directly here that you need to set the environment variable  `JAZZER_FUZZ` to `1`  to make it very clear and easier for beginners. Otherwise I fear that people will miss it. I know that it is mentioned a few paragraphs  down, but I think it is worth to mention it here as well.